Update website copy

jszwedko opened the pull request:
To copy provided in #199

Set asset_host so that assets linked in the e-mail have the full URL
(add staging environment since that asset_host is different).

Also remove Windows from the Gemfile.lock as Heroku will ignore the
Gemfile.lock in this case
(https://devcenter.heroku.com/articles/bundler-windows-gemfile)

jszwedko commented:
@FrankieBurton pushed this up to staging, it contains changes for:

#207 #206 #209 #204 #210 

(caveats as comments on the issues themselves)

Let me know what you think!

FrankieBurton commented:
Thanks @jszwedko! I may not be able to review these edits until tomorrow, but I will let you know what I think.

jszwedko commented:
@jeanwalshie @FrankieBurton merging this since all of the linked issues only had minor unaddressed feedback that I felt comfortable verifying.

This should close #207 #206 #209 #204 #210 #199 

Just let me know if you see any issues or want to make any other adjustments! Apologies for the delay, the holidays ended up busier than I anticipated.
